packages/template-retail-react-app/app/utils/routes-utils.test.js

```diff
@@ -288,4 +288,181 @@ describe('configureRoutes', function () {
             expect(paths).toEqual(expectedRes)
         })
     })
+
+    describe('fuzzyPathMatching', function () {
+        // Expected patterns based on sites mock:
+        // Sites: uk (alias) / site-1 (id), us (alias) / site-2 (id)
+        // Locales: en-GB, fr (alias), fr-FR, it-IT (site-1), en-US, en-CA (site-2)
+        const sitePattern = 'uk|site-1|us|site-2'
+        const localePattern = 'en-GB|fr|fr-FR|it-IT|en-US|en-CA'
```

Comment:
nit, what happens if there are invalid input like missing `sitePattern` or `localePattern`, does the code catch it and show warnings?

Reply:
As per my previous comment, these values are using the site and locale id's. If we want to, we can have a follow up to make sure that those values are validated, but that might be a breaking change ?